source/lex.h

```diff
@@ -930,26 +930,51 @@ auto lex_line(
         return 0;
     };
 
+    //G simple-hexadecimal-digit-sequence:
+    //G     hexadecimal-digit
+    //G     simple-hexadecimal-digit-sequence hexadecimal-digit
+    //G
     //G hexadecimal-escape-sequence:
     //G     '\x' hexadecimal-digit
     //G     hexadecimal-escape-sequence hexadecimal-digit
+    //G     '\x{' simple-hexadecimal-digit-sequence '}'
```

Comment:
Would it make sense to reuse `simple-hexadecimal-digit-sequence` for both cases?

    //G hexadecimal-escape-sequence:
    //G     '\x' simple-hexadecimal-digit-sequence
    //G     '\x{' simple-hexadecimal-digit-sequence '}'

Reply:
I wondered that too. I followed the adopted paper's wording, which didn't change the original productions.

However, now I see the that since then the standard did exactly what you suggest, so I'll do that too. Thanks!